user_guides/jaxws/src/main/jbake/content/config.inc

```diff
@@ -181,55 +250,33 @@ environment variables:
   d.  Set the `jaxws.classes` property to point to the Vendor
   Implementation classes/JAR files. +
   As an example, the `ts.jte` file contains the property
-  `jaxws.classes.ri`, which contains the classes/jar files that the Jakarta EE 9 Compatible Platform uses. The `jaxws.classes.ri` settings for the Jakarta EE {JakartaEEVersion} Compatible Platform web
-  container are as follows:
+  `jaxws.classes.ri`, which contains the classes/jar files that the Jakarta EE 9
```

Comment:
Consider changing the version to property (as above)

Reply:
OK